example.js

```diff
@@ -0,0 +1,32 @@
+// API proposal for Redux rest library
+
+// just usage examples
+
+import { rest } from '../lib/redux_api';
+
+// issue a get request
+const { user: { username }} = SETTINGS;
+rest.dashboard.get(username);
+
+rest.profile.get(username).then(profile => {
```

Comment:
Ah, I didn't consider that. Ideally we'd support that use-case from the beginning instead of needing to graft it on later, but I'm not sure exactly what the best way to do that would be.

We could maybe call `/api/v0/programs/` the `list` or `index` action, and reserve `get` for a detail view (i.e. for `/api/v0/programs/1`), hrrmmm

that would obviously get more complicated if the index view supported other verbs besides `GET`...

yeah, I'm leaning towards not handling that case for now, since we don't need it right now, and it wouldn't be too difficult to add it in later.